cassandra/protocol.py

```diff
@@ -129,25 +128,25 @@ class Codec(object):
         def __init__(self, error_decoders):
             self.error_decoders = {opcode: decoder for opcode, decoder in error_decoders}
 
-        def decode(self, f, protocol_version, *args):
+        def decode(self, f, protocol_version, *args, **kwargs):
```

Comment:
Here and elsewhere: why add `**kwargs` over a `protocol_version_registry` kwarg?

Reply:
It was mostly for the futur flexibility, but I have no objection to switch back to **kwargs if we need to if we hit a case.

Reply:
That sounds like a plan!